sycl/include/sycl/ext/oneapi/experimental/user_defined_reductions.hpp

```diff
@@ -41,6 +37,25 @@ reduce_over_group(GroupHelper group_helper, T x, BinaryOperation binary_op) {
                       PI_ERROR_INVALID_DEVICE);
 #endif
 }
+} // namespace detail
```

Comment:
Please, add `std::ignore = ` for `x`, `num_elements` and `binary_op` for host compilation.